### PR TITLE
Update deploy-python-model-quickstart.yaml

### PR DESCRIPTION
**Pull request by jjasghar**
## Description
- Updated to the "prediction_s_" not "prediction" per the flask app.
- Fixed the formating with "<" not being able to be read via the tutorial software.

## How Has This Been Tested?
Verified in my test env.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by andrewballantyne** [comment]
@kywalker-rh Not sure who reviews our Quick Starts -- but there is a slight change to the python one needing a content review.

**Comment by kywalker-rh** [comment]
> 

@andrewballantyne I'm not sure either. Maybe Audrey?

File: odh-dashboard/apps/jupyter/deploy-python-model-quickstart.yaml [hunk 1]
```diff
@@ -65,11 +65,11 @@ spec:
         3. In a Jupyter notebook, navigate to a terminal view.
         4. Run this curl command, using the URL you copied in step 1.
         ```
-        curl -X POST -d '{"hello" ":" "world"}' <URL>/prediction
+        curl -X POST -d '{"hello" ":" "world"}' YOUR_ROUTE/predictions
```
Comment (dchourasia) [inline odh-dashboard/apps/jupyter/deploy-python-model-quickstart.yaml:68]:
```suggestion
        curl -X POST -d '{"hello" : "world"}' YOUR_ROUTE/predictions
```

File: odh-dashboard/apps/jupyter/deploy-python-model-quickstart.yaml [hunk 2]
```diff
@@ -65,11 +65,11 @@ spec:
...
         ```
         For example:
         ```
-        curl -X POST -d '{"hello" ":" "world"}' http://example.apps.organization.abc3.p4.openshiftapps.com/prediction
+        curl -X POST -d '{"hello" ":" "world"}' http://example.apps.organization.abc3.p4.openshiftapps.com/predictions
```
Comment (dchourasia) [inline odh-dashboard/apps/jupyter/deploy-python-model-quickstart.yaml:72]:
```suggestion
        curl -X POST -d '{"hello" : "world"}' http://example.apps.organization.abc3.p4.openshiftapps.com/predictions
```

**Review by dchourasia — CHANGES_REQUESTED** [review]

**Comment by andrewballantyne** [comment]
/retest

Yeah, they gotta be a flake... you're changing yaml content. Let us retest and see.

**Comment by andrewballantyne** [comment]
@LaVLaS @VaishnaviHire is our tests really that flaky? I'm not sure how this PR could break a test, as it's just quickstart content.

**Comment by LaVLaS** [comment]
> @LaVLaS @VaishnaviHire is our tests really that flaky? I'm not sure how this PR could break a test, as it's just quickstart content.

Check the logs to see that the dashboard test is failing because the Jupyter tile is not visible when it attempts to spawn a notebook. See [Screenshots](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/opendatahub-io_odh-manifests/835/pull-ci-opendatahub-io-odh-manifests-master-411-odh-manifests-e2e/1666782927821737984/artifacts/odh-manifests-e2e/e2e/artifacts/ods-ci-2023-06-08-13-33-mQZNtPgk8A/) from the test run.  Without debugging the CI environment while it is running this PR should rebase on top of the latest dashboard version

**Comment by andrewballantyne** [comment]
Okay sweet... we got a success.

/lgtm

**Comment by andrewballantyne** [comment]
Awesome, thanks for your PR @jjasghar -- we got through the CI troubles 😄 this should merge automatically when the bot finishes up.

**Pull request merged**